clients/client-java/src/main/java/com/datastrato/gravitino/client/BaseSchemaCatalog.java

```diff
@@ -37,8 +37,9 @@
 abstract class BaseSchemaCatalog extends CatalogDTO implements Catalog, SupportsSchemas {
   /** The REST client to send the requests. */
   protected final RESTClient restClient;
+
   /** The namespace of current catalog, which is the metalake name. */
-  protected final Namespace namespace;
+  private final Namespace namespace;
```

Comment:
1. I think the `namespace` variable is not meaningful, either it can be catalogNamespace.
2. We have two `namespace`, one is catalog and another is schema, several information are shared, so instead of maintaining namespaces, we can maintain like metalake, catalog name directly and remove the duplication.

Reply:
For 1, good suggestion, I can quickly rename that.
For 2, using metalake name seems be a simple way. But as of present, those Builder classes (like `MessagingCatalog.Builder`, `RelationalCatalog.Builder`, etc) all use a "withNamespace()" method to contruct the object. So if the object just maintain a metalake name, these Builder classes should better be updated as well (e.g, "withMetalake()"), otherwise it will be odd. What do you think?

Reply:
Done, renamed to catalogNamespace